pkg/process/monitor/process_monitor_test.go

```diff
@@ -174,3 +177,29 @@ func TestProcessMonitorRefcount(t *testing.T) {
 		return numberOfExecs == oldNumberOfExecs
 	}, time.Second, time.Millisecond*200, fmt.Sprintf("capture a new exec %d old %d", numberOfExecs, oldNumberOfExecs))
 }
+
+func TestProcessMonitorInNamespace(t *testing.T) {
+	execSet := sync.Map{}
+
+	pm := GetProcessMonitor()
+	unsubscribeExec, err := pm.Subscribe(&ProcessCallback{
+		Event:    EXEC,
+		Metadata: ANY,
+		Callback: func(pid uint32) {
+			execSet.Store(pid, struct{}{})
+		},
+	})
+	defer unsubscribeExec()
+	require.NoError(t, err, "could not subscribe to EXEC events")
```

Comment:
on error, `unsubscribeExec` is nil
```suggestion
	require.NoError(t, err, "could not subscribe to EXEC events")
	defer unsubscribeExec()
```